updated missing release notes for semantic version in the UI component (#361)
SoftwareAG/c8y-releasenotes: content/release-10-18+/applications-10-18+.md

</tr>
</thead><tbody>

<tr>
<td>Cockpit</td>

> <td>In the "Data point graph" widget configuration, you could only select a chart type if the datapoint was not linked to a template from the data point library. Now you can always select a chart type.</td>

Reviewer: ```suggestion
<td>In the "Data point graph" widget configuration, you could only select a chart type if the data point was not linked to a template from the data point library. Now you can always select a chart type.</td>

```

<td>MTM-54007</td>
<td>10.18.497.13</td>
<td>UI</td>
</tr>

<tr>
<td>Cockpit</td>

> <td>If a user drags the X axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

Reviewer: ```suggestion
<td>If a user drags the X-axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

```

<tr>
<td>Web SDK</td>
<td>The export of the <code>@c8y/client</code> package was missing the required types which caused issues in certain setups. This has been adjusted.</td>
<td>MTM-56385</td>
<td>10.18.496.6</td>
<td>UI</td>
</tr>

<tr>
<td>Web SDK</td>

> <td>Fixed an issue with <code>c8ycli locale-extract</code> on Windows where the extracted <i>locales.pot</i> file didn´t contain any entries.</td>

Reviewer: ```suggestion
<td>Fixed an issue with <code>c8ycli locale-extract</code> on Windows where the extracted <i>locales.pot</i> file didn't contain any entries.</td>

```

<td>MTM-56234</td>
<td>10.18.496.1</td>
<td>UI</td>
</tr>

<tr>
<td>Web SDK</td>

> <td>After creating an asset with the Location property, where the default values for longitude and latitude are set, a map displaying a marker is displayed. If one or both of these values is deleted, the map will be hidden from the sub-assets view. When editing the Location property and these values are missing, the marker for selecting a location is not shown.</td>

Reviewer: ```suggestion
<td>After creating an asset with the Location property where the default values for longitude and latitude are set a map showing a marker is displayed. If one or both of these values is deleted, the map is hidden from the sub-assets view. When editing the Location property and these values are missing, the marker for selecting a location is not shown.</td>

```

<td>MTM-55865</td>
<td>10.18.495</td>
<td>UI</td>
</tr>

<tr>
<td>Web SDK</td>

> <td>If a filter was set in the extensions, applications or microservices lists, reloading the page on item removal or addition did not work. This issue has been fixed.</td>

Reviewer: maybe reverse order of "removal or addition" (-> "addition or removal")
